### ISSUE TITLE
research clearmash convert processor error (missing working_status field)

### ISSUE THREAD

**Issue by OriHoch** [Beit-Hatfutsot/mojp-dbs-pipelines#39]
error appeared in the pipelines dashboard for movies:
```
datapackage_pipelines_mojp.clearmash.processors.convert: INFO:base_processors:{'changeset': 5401740, 'collection': 'movies', 'document_id': '6A122325-1BB2-4871-A2A5-D44839F9D4F8', 'item_id': 960959, 'item_url': '', 'metadata': {'ActiveLock': None, 'CreatorUserId': 2, 'EntityTypeId': 49, 'IsArchived': False, 'IsBookmarked': False, 'IsLiked': False, 'LikesCount': 0}, 'parsed_doc': {'EntityFirstPublishDate': ['Datetime', {'Id': 'EntityFirstPublishDate', 'Value': {'UtcTicks': 636317947519000064}}], 'EntityLastPublishDate': ['Datetime', {'Id': 'EntityLastPublishDate', 'Value': {'UtcTicks': 636317947519000064}}], 'EntityViewsCount': 0, 'community_id': 6, 'entity_creation_date': ['Datetime', {'Id': 'entity_creation_date', 'Value': {'UtcTicks': 636317947519000064}}], 'entity_has_pending_changes': False, 'entity_id': 960959, 'entity_name': {'en': 'Gedcom'}, 'entity_type_id': 49, 'is_archived': False, 'is_deleted': False}, 'template_changeset_id': 5135675, 'template_id': '_90a9152316e34a10b7ef80cf367ee09e'}

datapackage_pipelines_mojp.clearmash.processors.convert: Traceback (most recent call last):
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/clearmash/processors/convert.py", line 49, in 
datapackage_pipelines_mojp.clearmash.processors.convert:     ClearmashConvertProcessor.main()
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/common/processors/base_processors.py", line 23, in main
datapackage_pipelines_mojp.clearmash.processors.convert:     spew(*cls(*ingest()).spew())
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/usr/local/lib/python3.6/site-packages/datapackage_pipelines/wrapper/wrapper.py", line 59, in spew
datapackage_pipelines_mojp.clearmash.processors.convert:     for rec in res:
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/clearmash/processors/convert.py", line 26, in _filter_resource
datapackage_pipelines_mojp.clearmash.processors.convert:     if self._doc_show_filter(dbs_row):
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/clearmash/processors/convert.py", line 18, in _doc_show_filter
datapackage_pipelines_mojp.clearmash.processors.convert:     return bool(dbs_row["source_doc"]["parsed_doc"].get('_c6_beit_hatfutsot_bh_base_template_working_status')[0].get("en") == 'Completed'
datapackage_pipelines_mojp.clearmash.processors.convert: TypeError: 'NoneType' object is not subscriptable
```

and similar error for personalities:
```
datapackage_pipelines_mojp.clearmash.processors.convert: Traceback (most recent call last):
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/clearmash/processors/convert.py", line 49, in 
datapackage_pipelines_mojp.clearmash.processors.convert:     ClearmashConvertProcessor.main()
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/common/processors/base_processors.py", line 23, in main
datapackage_pipelines_mojp.clearmash.processors.convert:     spew(*cls(*ingest()).spew())
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/usr/local/lib/python3.6/site-packages/datapackage_pipelines/wrapper/wrapper.py", line 59, in spew
datapackage_pipelines_mojp.clearmash.processors.convert:     for rec in res:
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/clearmash/processors/convert.py", line 26, in _filter_resource
datapackage_pipelines_mojp.clearmash.processors.convert:     if self._doc_show_filter(dbs_row):
datapackage_pipelines_mojp.clearmash.processors.convert:   File "/mojp/datapackage_pipelines_mojp/clearmash/processors/convert.py", line 18, in _doc_show_filter
datapackage_pipelines_mojp.clearmash.processors.convert:     return bool(dbs_row["source_doc"]["parsed_doc"].get('_c6_beit_hatfutsot_bh_base_template_working_status')[0].get("en") == 'Completed'
datapackage_pipelines_mojp.clearmash.processors.convert: TypeError: 'NoneType' object is not subscriptable
```

**Comment by OriHoch** [comment]
there is a general problem with the error log - it doesn't provide enough details, so hard to debug

**Comment by OriHoch** [comment]
the "bad" movie:
```js
{
  'changeset': 5401740,
  'collection': 'movies',
  'document_id': '6A122325-1BB2-4871-A2A5-D44839F9D4F8',
  'item_id': 960959,
  'item_url': '',
  'metadata': {
    'ActiveLock': None,
    'CreatorUserId': 2,
    'EntityTypeId': 49,
    'IsArchived': False,
    'IsBookmarked': False,
    'IsLiked': False,
    'LikesCount': 0
  },
  'parsed_doc': {
    'EntityFirstPublishDate': [
      'Datetime',
      {
        'Id': 'EntityFirstPublishDate',
        'Value': {
          'UtcTicks': 636317947519000064
        }
      }
    ],
    'EntityLastPublishDate': [
      'Datetime',
      {
        'Id': 'EntityLastPublishDate',
        'Value': {
          'UtcTicks': 636317947519000064
        }
      }
    ],
    'EntityViewsCount': 0,
    'community_id': 6,
    'entity_creation_date': [
      'Datetime',
      {
        'Id': 'entity_creation_date',
        'Value': {
          'UtcTicks': 636317947519000064
        }
      }
    ],
    'entity_has_pending_changes': False,
    'entity_id': 960959,
    'entity_name': {
      'en': 'Gedcom'
    },
    'entity_type_id': 49,
    'is_archived': False,
    'is_deleted': False
  },
  'template_changeset_id': 5135675,
  'template_id': '_90a9152316e34a10b7ef80cf367ee09e'
}
```

the "bad" personality:
```js
{
  'changeset': 5402788,
  'collection': 'personalities',
  'document_id': '660BE334-075A-4A40-94C2-AAB6FEA38BBD',
  'item_id': 266860,
  'item_url': '',
  'metadata': {
    'ActiveLock': None,
    'CreatorUserId': 2,
    'EntityTypeId': 49,
    'IsArchived': False,
    'IsBookmarked': False,
    'IsLiked': False,
    'LikesCount': 0
  },
  'parsed_doc': {
    'EntityFirstPublishDate': [
      'Datetime',
      {
        'Id': 'EntityFirstPublishDate',
        'Value': {
          'UtcTicks': 636310649209833344
        }
      }
    ],
    'EntityLastPublishDate': [
      'Datetime',
      {
        'Id': 'EntityLastPublishDate',
        'Value': {
          'UtcTicks': 636310649209833344
        }
      }
    ],
    'EntityViewsCount': 7,
    'community_id': 6,
    'entity_creation_date': [
      'Datetime',
      {
        'Id': 'entity_creation_date',
        'Value': {
          'UtcTicks': 636310649209833344
        }
      }
    ],
    'entity_has_pending_changes': False,
    'entity_id': 266860,
    'entity_name': {
      'en': 'Gedcom'
    },
    'entity_type_id': 49,
    'is_archived': False,
    'is_deleted': False
  },
  'template_changeset_id': 5135675,
  'template_id': '_90a9152316e34a10b7ef80cf367ee09e'
}
```

**Comment by OriHoch** [comment]
fixed in 0.1.2
